src/pages/workspace/reimburse/WorkspaceReimburseSection.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ActivityIndicator, View} from 'react-native';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import Text from '../../../components/Text';
+import styles from '../../../styles/styles';
+import themeColors from '../../../styles/themes/default';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import * as Illustrations from '../../../components/Icon/Illustrations';
+import Section from '../../../components/Section';
+import * as Link from '../../../libs/actions/Link';
+import Button from '../../../components/Button';
+import BankAccount from '../../../libs/models/BankAccount';
+import reimbursementAccountPropTypes from '../../ReimbursementAccount/reimbursementAccountPropTypes';
+import * as ReimbursementAccount from '../../../libs/actions/ReimbursementAccount';
+import networkPropTypes from '../../../components/networkPropTypes';
+import CONST from '../../../CONST';
+
+const propTypes = {
+    /** Policy values needed in the component */
+    policy: PropTypes.shape({
+        id: PropTypes.string,
+    }).isRequired,
+
+    /** Bank account attached to free plan */
+    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
+
+    /** Information about the network */
+    network: networkPropTypes.isRequired,
+
+    /** Returns translated string for given locale and phrase */
+    translate: PropTypes.func.isRequired,
+};
+
+class WorkspaceReimburseSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            shouldShowLoadingSpinner: false,
+        };
+
+        this.debounceSetShouldShowLoadingSpinner = _.debounce(this.setShouldShowLoadingSpinner.bind(this), CONST.TIMING.SHOW_LOADING_SPINNER_DEBOUNCE_TIME);
+    }
+
+    componentDidUpdate() {
+        this.debounceSetShouldShowLoadingSpinner();
+    }
+
+    setShouldShowLoadingSpinner() {
+        this.setState({shouldShowLoadingSpinner: this.props.reimbursementAccount.isLoading || false});
+    }
+
+    render() {
+        const achState = lodashGet(this.props.reimbursementAccount, 'achData.state', '');
+        const hasVBA = achState === BankAccount.STATE.OPEN;
+
+        if (this.props.network.isOffline) {
```

Comment:
hi all
we forgot to add offline behavior for other pages that had connect bank account button
eg: Cards’, ‘Bills’, ‘Invoices’, ‘Travel’ etc.

because of the missed case, we created an issue here - https://github.com/Expensify/App/issues/21186